acs-integration-tests/pom.xml

```diff
@@ -420,19 +433,10 @@
                             <summaryFile>${basedir}/failsafe-reports-predix-cloud-titan/failsafe-summary.xml</summaryFile>
                             <systemPropertyVariables>
                                 <ACS_CF_DOMAIN>${acs.cf.domain}</ACS_CF_DOMAIN>
-                                <ACS_DEFAULT_ISSUER_ID>${ACS_DEFAULT_ISSUER_ID}</ACS_DEFAULT_ISSUER_ID>
-                                <ACS_SERVICE_ID>predix-acs</ACS_SERVICE_ID>
                                 <ACS_URL>https://${acs.appname}.${cf.base.domain}</ACS_URL>
```

Comment:
`ACS_URL` should be moved out to the `systemPropertyVariables` section of the common `maven-failsafe-plugin` configuration you have towards the beginning of the file (since it's also common to every profile)